Add multi-node administration section (#862)
timescale/docs: timescaledb/how-to-guides/multinode-timescaledb/multinode-administration.md

connect from the access node to a data node as that user, the data
nodes need to have the role configured with the `LOGIN` attribute
enabled. To create a non-login role for a multi-node setup, use these
commands:
```sql
CREATE ROLE alice WITHOUT LOGIN;
CALL distributed_exec($$ CREATE ROLE alice WITH LOGIN; $$);
```
To allow a new role to create distributed hypertables it also needs to
be granted usage on data nodes, for example:
```sql
GRANT USAGE ON FOREIGN SERVER dn1,dn2,dn3 TO alice;
```

By granting usage on some data nodes, but not others, you can
restrict usage to a subset of data nodes based on the role.

### Alter a distributed role
When you alter a distributed role, use the same process as creating
roles. The role needs to be altered on the access node and on the data
nodes in two separate steps. For example, add the `CREATEROLE`
attribute to a role as follows:
```sql
ALTER ROLE alice CREATEROLE;
CALL distributed_exec($$ ALTER ROLE alice CREATEROLE; $$);
```

## Manage distributed databases
A distributed database can contain both distributed and
non-distributed objects. In general, when a command is issued to alter

> a distributed object it applies to all nodes that have that object (or

Reviewer: ```suggestion
a distributed object, it applies to all nodes that have that object (or
```

a part of it).

However, in some cases settings *should* be different depending on

> node, because nodes might be provisioned differently (having, e.g.,

Reviewer: ```suggestion
node, because nodes might be provisioned differently (having, for example,
```

to drop a database even if one or more data nodes are not responding.

It is also good practice to leave the data intact on a data node if
possible. For example, you might want to back up a data node even
after a database was dropped on the access node.

Alternatively, you can delete the data nodes with
the `drop_database` option prior to dropping the database on the
access node:
```sql
SELECT * FROM delete_data_node('dn1', drop_database => true);
```

## Create, alter, and drop schemas
When you create, alter, or drop schemas, the commands are not
automatically applied across all data nodes. A missing schema is,
however, created when a distributed hypertable is created, and the
schema it belongs to does not exist on a data node.

To manually create a schema across all data nodes, use the following
command:
```sql
CREATE SCHEMA newschema;
CALL distributed_exec($$ CREATE SCHEMA newschema $$);
```

If a schema is created with a particular authorization, then the
authorized role must also exist on the data nodes prior to issuing the
command. The same things applies to altering the owner of an existing
schema.

### Prepare for role removal with DROP OWNED

> The [`DROP OWNED`][drop-owned] command is used drop all objects owned

Reviewer: ```suggestion
The [`DROP OWNED`][drop-owned] command is used to drop all objects owned
```
